engine/updategraph/src/main/java/io/deephaven/engine/updategraph/UpdateGraphProcessor.java

```diff
@@ -276,7 +276,7 @@ public synchronized void take(final AccumulatedCycleStats out) {
         refreshThread.setDaemon(true);
 
         final int updateThreads =
-                Configuration.getInstance().getIntegerWithDefault("UpdateGraphProcessor.updateThreads", -1);
+                Configuration.getInstance().getIntegerWithDefault("UpdateGraphProcessor.updateThreads", 1);
```

Comment:
This doesn't look right, it reverses the recent change to multi-thread by default.